src/main/scala/chisel3/simulator/SimulatorAPI.scala

```diff
@@ -102,4 +103,84 @@ trait SimulatorAPI {
     stimulus(dut)
   }
 
+  def simulateTests[T <: RawModule with HasTests](
+    module:       => T,
+    testGlobs:    Seq[String],
+    timeout:      Int,
+    chiselOpts:   Array[String] = Array.empty,
+    firtoolOpts:  Array[String] = Array.empty,
+    settings:     Settings[TestHarness[T, _]] = Settings.defaultRaw[TestHarness[T, _]],
+    subdirectory: Option[String] = None
+  )(
+    implicit hasSimulator:        HasSimulator,
+    testingDirectory:             HasTestingDirectory,
+    chiselOptsModifications:      ChiselOptionsModifications,
+    firtoolOptsModifications:     FirtoolOptionsModifications,
+    commonSettingsModifications:  svsim.CommonSettingsModifications,
+    backendSettingsModifications: svsim.BackendSettingsModifications
+  ): Seq[(String, Simulator.BackendInvocationDigest[_])] = {
```

Comment:
The return type of these methods is too low-level. This should just be `Unit`. Alternatively, there should be a `ChiselSim`-specific return type that is used for _all_ methods here. I.e., also for `simulate` and `simulateRaw`.